docs/core/tools/dotnet-publish.md

```diff
@@ -61,6 +61,10 @@ Specifies the path for the output directory. If not specified, it defaults to *.
 
 Configuration to use when building the project. The default value is `Debug`.
 
+`--manifest <PATH_TO_MANIFEST_FILE>`
```

Comment:
There is no syntax for 1.0; so in the paragraph below, I put ...

> This option is available starting with .NET Core SDK 2.0 Preview 2.

If I remove it, then we should open an issue to add it back when 2.0 ships. Let me know which way to go.

RE: Tabs -- I'm not sure how the tabs work tho when the prior version doesn't have the feature at all. I guess the 1.0 tab can say something to the effect, "This option isn't available in 1.0."